### PR TITLE
[hi] Localize dockershim.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/reference/glossary/dockershim.md` to Hindi `content/hi/docs/reference/glossary/dockershim.md`

Fixes :- #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/dockershim.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+full_link: /dockershim
+short_description: >
+  कुबेरनेट्स v1.23 और इससे पहले का एक घटक, जो कुबेरनेट्स सिस्टम घटकों को डोकर इंजन के साथ संचार करने की अनुमति देता है।
+
+aka:
+tags:
+  - fundamental
+---
+
+डोकरशिम कुबेरनेट्स संस्करण 1.23 और इससे पहले का एक घटक है। यह क्यूबलेट को {{<glossary_tooltip text="डोकर इंजन" term_id="docker" >}} के साथ संचार करने के लिए अनुमति देता है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/dockershim.md:14]:
```suggestion
डॉकरशिम कुबेरनेट्स संस्करण 1.23 और इससे पहले का एक घटक है। यह क्यूबलेट को {{<glossary_tooltip text="डॉकर इंजन" term_id="docker" >}} के साथ संचार करने के लिए अनुमति देता है।
```

File: content/hi/docs/reference/glossary/dockershim.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+<!--more-->
+
+संस्करण 1.24 से शुरू होकर, डोकरशिम को कुबेरनेट्स से हटा दिया गया है। अधिक जानकारी के लिए, [डॉकरशिम FAQ](/dockershim) देखें।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/dockershim.md:18]:
```suggestion
संस्करण 1.24 से शुरू होकर, डॉकरशिम को कुबेरनेट्स से हटा दिया गया है। अधिक जानकारी के लिए, [डॉकरशिम FAQ](/dockershim) देखें।
```

**Review by Babapool — COMMENTED** [review]
Some spelling nits:

**Comment by Babapool** [comment]
The changes look LGTM ( I am not official 'hi' localization reviewer). Please squash your commits into one commit to make your PR ready to merge. Thanks.

File: content/hi/docs/reference/glossary/dockershim.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: डोकरशिम (Dockershim)
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/dockershim.md:2]: I think phonetically, डॉ sounds closer to the English word than डो
```suggestion
title: डॉकरशिम (Dockershim)
```

File: content/hi/docs/reference/glossary/dockershim.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+id: dockershim
+date: 2022-04-15
+full_link: /dockershim
+short_description: >
+  कुबेरनेट्स v1.23 और इससे पहले का एक घटक, जो कुबेरनेट्स सिस्टम घटकों को डॉकर इंजन के साथ संचार करने की अनुमति देता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/dockershim.md:7]:
```suggestion
  कुबेरनेट्स संस्करण 1.23 और पहले का एक घटक, जो कुबेरनेट्स सिस्टम घटकों को डॉकर इंजन के साथ संचार करने की अनुमति देता है।
```

File: content/hi/docs/reference/glossary/dockershim.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+aka:
+tags:
+  - fundamental
+---
+
+डॉकरशिम कुबेरनेट्स संस्करण 1.23 और इससे पहले का एक घटक है। यह क्यूबलेट को {{<glossary_tooltip text="डॉकर इंजन" term_id="docker" >}} के साथ संचार करने के लिए अनुमति देता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/dockershim.md:14]:
```suggestion
डॉकरशिम कुबेरनेट्स संस्करण 1.23 और पहले का एक घटक है। यह क्यूबलेट को {{<glossary_tooltip text="डॉकर इंजन" term_id="docker" >}} के साथ संचार करने के लिए अनुमति देता है।
```

**Review by divya-mohan0209 — DISMISSED** [review]
Some inconsistency nits. Rest LGTM!

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Review by Babapool — COMMENTED** [review]
LGTM from my side as well. Thanks
/lgtm

**Comment by divya-mohan0209** [comment]
Using [the comment](https://github.com/kubernetes/website/pull/36323#pullrequestreview-1227205687) from @Babapool to provide approval for merging.
/approve

**Pull request merged**